material-overrides/assets/stylesheets/moonbeam.css

```diff
@@ -2176,3 +2176,20 @@ ul.task-list {
     display: none;
   }
 }
+
+/*Responsive style for wide screen footer, header, consent*/
+@media screen and (min-width: 2560px) {
+  .md-header__inner.md-grid,
+  .md-footer__inner.md-grid{
+      max-width: 80rem; 
+
+  }
+  form.md-consent__form.md-grid.md-typeset {
+    max-width: 80rem; 
+    margin: 2em auto !important; 
+  }
+  .md-banner__button {
+    right: 17em;
+    position: absolute;
+  }
```

Comment:
Not sure about this, I feel like it looks better with the button closer to the text 🤷‍♀️ 

If you want it to line up more with the max-width of the page, you could remove these `.md-banner_button` styles and try adding:

```
.md-banner__inner.md-grid.md-typeset {
    max-width: 80rem;
}
```